src/absences/types.ts

```diff
@@ -62,3 +62,19 @@ export type Absence = {
 };
 
 export interface AbsenceCreate extends Omit<Absence, 'id' | 'created' | 'status' | 'duration' | 'duration_unit' | 'updated'> {}
+
+export interface AbsenceApprove {
+  id: number;
+  granted_comment?: string;
+  substitute_enabled?: boolean;
+}
+
+export interface AbsenceReject extends AbsenceApprove {}
+
+export interface AbsenceCancel {
```

Comment:
When absenceCancel only has an id as a field, we can remove this type and to the same, as we have for delete.. call the endpoint with a number as an argument.

Reply:
done